Arrays-Implementation/subarray/subsetOfArray.cpp

```diff
@@ -0,0 +1,23 @@
+//Given two arrays: a1[0..n-1] of size n and a2[0..m-1] of size m. Task is to check whether a2[] is a subset of a1[] or not.
+//Both the arrays can be sorted or unsorted. It may be assumed that elements in both array are distinct.
+ 
+
+string isSubset(int a1[], int a2[], int n, int m) {
```

Comment:
if the name of the function is `isSubset` then your return type should be **_bool_**
pls check and change the return type.